doc/build.md

```diff
@@ -2,17 +2,23 @@
 
 ## Required Components
 You will need the following tools and components to build the Basketweave-S (not included in the kit):
+![components](https://imgur.com/C8kzkQa.png)
 - Soldering iron and solder
 - Phillips head screwdriver
 - Flush side cutters
-- No-clean flux (recommended to prevent bridging on USB pins)
-- 70 MX style switches (or 71 when not using the encoder)
-- 5 PCB mount 2u stabilizers
+- No-clean flux (heavily recommended to prevent bridging on USB pins)
+- 69-72 MX style switches (1 less per encoder, 1 more if you use split backspace)
```

Comment:
Or maybe "71 -1 for each encoder, +1 for split backspace" in brackets?

(In any case, it's "fewer" for countable words ;)